src/App.tsx

```diff
@@ -10,12 +10,14 @@ import RescheduleAppointment from './pages/appointments/RescheduleAppointment';
 import SummaryAppointment from './pages/appointments/SummaryAppointment';
 import UnAuthorized from './pages/errors/UnAuthorized';
 import NotFound from './pages/errors/NotFound';
+import { SECONDARY_COLOR } from './utils/ConstantsColors';
+
 const App = () => {
   return (
     <ConfigProvider
       theme={{
         token: {
-          colorPrimary: '#1e40af',
+          colorPrimary: SECONDARY_COLOR,
```

Comment:
why constant SECONDARY_COLOR is the colorPrimary? I mean should be the same one right?

Reply:
@imadiaz Ok, I have already corrected it